mqtt_client:
  ros__parameters:
    broker:
      host: emqx
      port: 1883
    bridge:
      ros2mqtt:
        ros_topics: 
          - /ping/ros
        /ping/ros:
          mqtt_topic: pingpong/ros
      mqtt2ros:
        mqtt_topics: 
          - pingpong/ros
        pingpong/ros:
          ros_topic: /pong/ros
